README.md

```diff
@@ -5,102 +5,165 @@
 It runs with [Prow CI](https://github.com/kubernetes/test-infra/blob/master/prow/) on a [Google Kubernetes Engine Cluster](https://cloud.google.com/kubernetes-engine/).
 It is designed to support adding more k8s providers.
 
+## How to run tests manually
+
+### Prerequisites
+---
+
+#### Create a k8s cluster
+
+- Create a new project on Google Cloud.
+
+- Create a Service Account on GKE with role `Kubernetes Engine Service Agent` & `Kubernetes Engine Admin` and download the json file.
+
+- Set the following environment variables & deploy cluster.
+```
+export PROJECT_ID=<google-cloud project-id>
+export CLUSTER_NAME=prombench
+export ZONE=us-east1-b
+export AUTH_FILE=<path to service-account.json>
+
+./prombench gke cluster create -a $AUTH_FILE -v PROJECT_ID:$PROJECT_ID \
+    -v ZONE:$ZONE -v CLUSTER_NAME:$CLUSTER_NAME -f components/prow/cluster.yaml
+```
+
+### Deploy Prometheus-Meta & Grafana
+---
+
+This is used for collecting and displaying the test results.
```

Comment:
lets have this as a quote before --- , it is less distracting and better suited for a note:
> for collecting and displaying the test results.